integreat_cms/cms/views/analytics/translation_coverage_view.py

```diff
@@ -44,67 +45,48 @@ def get(self, request, *args, **kwargs):
 
         region = request.region
 
-        total_number_pages = region.pages.count()
-
-        translation_coverage_data = []
-        outdated_word_count = {}
+        translation_coverage_data = {}
+        outdated_word_count = Counter()
 
+        pages = [
+            page
+            for page in region.pages.filter(explicitly_archived=False)
+            .prefetch_translations()
+            .cache_tree()
+            if not page.archived
+        ]
```

Comment:
```suggestion
        pages = (
            region.pages.filter(explicitly_archived=False)
            .prefetch_translations()
            .cache_tree()
        )
```
Actually, `cache_tree()` already skips nodes that don't have a parent node in the queryset, so if it's filtered by `explicitly_archived=False`, we don't have to perform additional filters.